ui/src/lib/features/k8s/storage/persistentvolumes/store.ts

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024-Present The UDS Authors
+
+import type { V1PersistentVolume as Resource } from '@kubernetes/client-node'
+
+import { ResourceStore, transformResource } from '$features/k8s/store'
+import { type ColumnWrapper, type CommonRow, type ResourceStoreInterface } from '$features/k8s/types'
+
+interface Row extends CommonRow {
+  storage_class: string
+  capacity: string
+  claim: string
+  status: string
+}
+
+export type Columns = ColumnWrapper<Row>
+
+export function createStore(): ResourceStoreInterface<Resource, Row> {
+  const url = `/api/v1/resources/storage/persistentvolumes`
```

Comment:
```suggestion
  const url = `/api/v1/resources/storage/persistentvolumes?dense=true`
```
adds missing fields that are now in dense